elisp/core.el

```diff
@@ -217,4 +217,6 @@
                             mal-nil))
                          (t
                           mal-nil))))))
+
+    (elisp-eval . ,(mal-fn (lambda (string) (mal-string (format "%S" (eval (read (mal-value string))))))))
     ))
```

Comment:
What should really be happening here is that the resulting elisp data types are converted to equivalent mal data types instead of just returning a string representation. Maybe just removing the (mal-string (format)) would do the trick?